build-tools-internal/src/main/groovy/elasticsearch.build-complete.gradle

```diff
@@ -9,8 +9,8 @@
 import java.nio.file.Files
 
 String buildNumber = System.getenv('BUILD_NUMBER')
-
-if (buildNumber) {
+String performanceTest = System.getenv('BUILD_PERFORMANCE_TEST')
```

Comment:
we do not want to generate the tar.gz2 at the end of each build. we manually tar up the profile output instead and upload the results to google cloud